src/components/Course.vue

```diff
@@ -23,16 +23,7 @@
           <span v-if="!compact && semesterString" class="course-semesters">{{
             semesterString
           }}</span>
-          <!-- <div v-if="!compact && alerts.requirement" class="course-outerWrapper course-tooltip">
-            <div class="course-iconWrapper course-iconWrapper--info">
-              <img class="course-icon course-icon--info" src="../assets/images/info.svg" />
-            </div>
-            <div
-              class="course-tooltiptext course-tooltiptext--info"
-              v-html="requirementString"
-            ></div>
-          </div> -->
```

Comment:
This segment of code and the related `loadRequirementsMap` has been dead for quite a while since https://github.com/cornell-dti/course-plan/commit/44c0a27ead2297dd8cc058a147bdab0aed52fb2b

I decided to remove it to make my refactor easier. It's also not a good idea to keep dead code around